pkg/authenticator/authenticator.go

```diff
@@ -133,6 +136,13 @@ func (auth *Authenticator) Login() error {
 		return log.RecordedError(log.CAKC029E, err.Error())
 	}
 
+	// ensure client certificate exists before attempting to read it, with a tolerance
+	// for small delays
+	err = auth.WaitForInjectedCertificateFile(auth.Config.ClientCertPath, auth.Config.ClientCertRetryCountLimit, os.Stat, os.IsNotExist)
```

Comment:
Line-too-long

Reply:
would you prefer it like this?
```
	err = auth.WaitForInjectedCertificateFile(
		auth.Config.ClientCertPath,
		auth.Config.ClientCertRetryCountLimit,
		os.Stat,
		os.IsNotExist)
```
i don't like when function signatures span on more than one line.

Reply:
> i don't like when function signatures span on more than one line.

Me neither but long lines make maintenance much harder.

I don't remember well but I think this is what we used for the hanging lines:
```go
err = auth.WaitForInjectedCertificateFile(
    auth.Config.ClientCertPath,
    auth.Config.ClientCertRetryCountLimit,
    os.Stat,
    os.IsNotExist,
 )
```

and https://github.com/cyberark/secretless-broker/blob/master/internal/plugin/connectors/http/generic/connector.go#L62-L65 for the definitions

Reply:
ok this is actually the call but same question for the function signature

Reply:
do we understand well how adding this wait will interact with the exponential backoff in main.go?

Reply:
why will it matter? the exponential backoff has some calculated interval between each try, no matter how long that try had taken. am i wrong? do you have other concerns?